MCP.md

```diff
@@ -84,19 +82,18 @@ This is to define expressions for `F` in MCP. This is merely an alias of `JuMP.@
 ```
 This macro matches each element of `F` and the complementing element of `x`.
 
-```julia
-PATHSolver.options(convergence_tolerance=1e-8, output=:yes, time_limit=3600)
-```
-This adjusts options of the PATH Solver. See the [list of options](http://www.cs.wisc.edu/~ferris/path/options.pdf).
 
 ```julia
-solveMCP(m)
+solveMCP(m; convergence_tolerance=1e-8, output="yes", time_limit=3600)
 ```
-This solves the MCP and stores the solution inside `m`, which can be accessed by `getvalue(x)` as in JuMP.
+This solves the MCP and stores the solution inside `m`, which can be accessed by `result_value(x)`.
+Keyword arguments are options of the PATH Solver. See the [list of options](http://www.cs.wisc.edu/~ferris/path/options.pdf).
 
 
 ## Linear Complementarity Problems
 
+**NOT Supported with the new PATHSolver**
```

Comment:
Why have a special `linear = true`?

Reply:
Sometime ago, people wanted to make PATHSolver.jl speedy for LCP by not calculating Jacobian multiple times. So it had `solveLCP` in addition to `solveMCP`, and `linear=true` is in the same line.

Reply:
I'd just remove it until people complain again. If people are solving linear problems and care that much, they can use `PATHSolver` directly.

Reply:
Sounds fair.

Reply:
Also, if I understand things correctly, for linear problems one can now use JuMP directly, right? So for linear problems Complementarity.jl isn't really needed?

Reply:
@davidanthoff Correct.